handlers/deployment.rb

```diff
@@ -55,28 +74,7 @@ class Deployment < Handler
       # state: the default deploy "chat ops" deploy system
       #        and in use for all K8s deployed services
       route(/^(deploy)\s*(.*)/, :tag_deploy, command: true, help: {"deploy REPO" => "Updates the production-release tag on zooniverse/REPO"})
-
-      def status(response)
-        deployed_version = HTTParty.get("https://panoptes.zooniverse.org/commit_id.txt").strip
-
-        git_responses = {}
-        %w(HEAD production-release).each do |tag|
-          comparison = Octokit.compare("zooniverse/panoptes", deployed_version, tag)
-          if comparison.commits.empty?
-            git_responses[tag] = "is the currently deployed version."
-          else
-            word = comparison.commits.size > 1 ? "commits" : "commit"
-            git_responses[tag] = "#{comparison.commits.size} undeployed #{word}. #{comparison.permalink_url}"
-            git_responses[tag] << " :shipit:" if tag == "production-release"
-          end
-        end
-
-        formatted_response = git_responses.map do |tag, comment|
-          "#{tag.upcase} : #{comment}"
-        end.join("\n")
-
-        response.reply(formatted_response)
-      end
+      route(/^(status|version)\s*(.*)/, :status, command: true, help: {'status GITHUB_ORG/REPO_NAME' => 'Returns the number of commits not deployed to production-release for the org/repo_name.'})
```

Comment:
Could we default the org name to Zooniverse if it's not specified? Or at least return an error message otherwise. I expected this to work:

```
Lita > lita status caesar
[2020-02-20 16:44:03 UTC] ERROR: Lita::Handlers::Deployment crashed. The exception was:
Failed to open TCP connection to .zooniverse.org:443 (getaddrinfo: Name or service not known)
Full backtrace:
/usr/local/lib/ruby/2.6.0/net/http.rb:949:in `rescue in block in connect'
/usr/local/lib/ruby/2.6.0/net/http.rb:946:in `block in connect'
/usr/local/lib/ruby/2.6.0/timeout.rb:93:in `block in timeout'
/usr/local/lib/ruby/2.6.0/timeout.rb:103:in `timeout'
/usr/local/lib/ruby/2.6.0/net/http.rb:945:in `connect'
/usr/local/lib/ruby/2.6.0/net/http.rb:930:in `do_start'
/usr/local/lib/ruby/2.6.0/net/http.rb:919:in `start'
/usr/local/lib/ruby/2.6.0/net/http.rb:1470:in `request'
/usr/local/bundle/gems/httparty-0.16.4/lib/httparty/request.rb:146:in `perform'
/usr/local/bundle/gems/httparty-0.16.4/lib/httparty.rb:573:in `perform_request'
/usr/local/bundle/gems/httparty-0.16.4/lib/httparty.rb:491:in `get'
/usr/local/bundle/gems/httparty-0.16.4/lib/httparty.rb:623:in `get'
/app/handlers/deployment.rb:233:in `get_deployed_commit'
/app/handlers/deployment.rb:206:in `get_repo_status'
/app/handlers/deployment.rb:150:in `status'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/callback.rb:31:in `public_send'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/callback.rb:31:in `call'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/handler/chat_router.rb:97:in `dispatch_to_route'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/handler/chat_router.rb:82:in `block in dispatch'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/handler/chat_router.rb:72:in `map'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/handler/chat_router.rb:72:in `dispatch'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/robot.rb:69:in `block in receive'
/usr/local/lib/ruby/2.6.0/set.rb:338:in `each_key'
/usr/local/lib/ruby/2.6.0/set.rb:338:in `each'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/robot.rb:66:in `map'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/robot.rb:66:in `receive'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/adapters/shell.rb:94:in `block in run_loop'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/adapters/shell.rb:85:in `loop'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/adapters/shell.rb:85:in `run_loop'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/adapters/shell.rb:34:in `run'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/robot.rb:80:in `run'
/usr/local/bundle/gems/lita-4.7.1/lib/lita.rb:80:in `run'
/usr/local/bundle/gems/lita-4.7.1/lib/lita/cli.rb:77:in `start'
/usr/local/bundle/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
/usr/local/bundle/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
/usr/local/bundle/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
/usr/local/bundle/gems/thor-0.20.0/lib/thor/base.rb:466:in `start'
/usr/local/bundle/gems/lita-4.7.1/bin/lita:6:in `<top (required)>'
/usr/local/bundle/bin/lita:23:in `load'
/usr/local/bundle/bin/lita:23:in `<main>'
```

Reply:
yeah - i was thinking that as i re-read the PR code. Can do.